apps/ui/src/Widget.tsx

```diff
@@ -50,10 +53,10 @@ const Widget = ({
           };
 
           if (window.parent) {
-            window.parent.postMessage(payload);
+            window.parent.postMessage(payload, refererUrl || '*');
```

Comment:
This is a security vulnerability, can we not default to `'*'`? If we're opening a security window like that, I'd like it to be explicitly set by the client.